### PR TITLE
GEOT-5120 Avoid querying areas on the other side of the dateline, whenn possible and wrapping is disabled

### PR DESCRIPTION
**Pull request by aaime**

### CODE REVIEW

File: modules/library/render/src/main/java/org/geotools/renderer/crs/ProjectionHandlerFinder.java [hunk 1]
```diff
@@ -84,7 +84,7 @@ public void setWrapLimit(int wrapLimit) {
     public static ProjectionHandler getHandler(ReferencedEnvelope renderingArea, CoordinateReferenceSystem sourceCrs, boolean wrap) throws FactoryException {
         if (renderingArea.getCoordinateReferenceSystem() == null)
             return null;
-
+
```
Comment (tbarsballe) [inline modules/library/render/src/main/java/org/geotools/renderer/crs/ProjectionHandlerFinder.java]: Bit of a nitpick, but this file only contains whitespace changes and should not be included in the commit.

**Comment by jodygarnett** [comment]
Merged to master as c9ea50f5c279711e2f69b1313535a173c3d37fc2

**Comment by jodygarnett** [comment]
Have to reopen this one, it caused failures in GeoSever org.geoserver.wcs2_0.xml.GetCoverageTest.testDatelineCrossingMercatorPDC due to generated envelope not match what was expected.
